os/drivers/lcd/mipi_lcd.c

```diff
@@ -38,6 +38,62 @@
 #define CONFIG_LCD_MAXPOWER 100
 #endif
 
+#if defined(CONFIG_LCD_SW_ROTATION)
+#define NUM_OF_LCD_BUFFER	2
+uint8_t *lcd_buffer[NUM_OF_LCD_BUFFER] = { NULL, NULL };	//Two lcd buffers to avoid screen tearing
+int lcd_buffer_index = 0;
```

Comment:
Maybe they are used in this file only. Let's add `static`.

Reply:
done